swarmrl/networks/flax_network.py

```diff
@@ -133,8 +133,7 @@ def compute_action(self, feature_vector: np.ndarray, explore_mode: bool = False)
 
         if explore_mode:
             index = self.exploration_policy(index, len(model_output))
-
-        return index, model_output[index]
+        return index, jax.nn.softmax(model_output)[index]
```

Comment:
This will break the normal loss right?

Reply:
You mean it would break the Policy Gradient Loss? 
As far as I can see, no. 
In policy gradient loss we don't use this information. We only give it the chosen action.